src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -114,6 +114,16 @@
             "type": "boolean",
             "description": "Whether the user saved"
         },
+        {
+            "name": "resourceCounts",
+            "type": "string",
```

Comment:
Not sure about the best way to do this, but do we have a way to extract this information in Kibana into readable graphs, given that this is a string type?

Reply:
That is a Carl question, but I'll follow up with him